doc/modules/gaussian_process.rst

```diff
@@ -602,16 +602,27 @@ References
 ----------
 
     * `[RW2006]
-      <http://www.gaussianprocess.org/gpml/chapters/>`_ 
-      **Gaussian Processes for Machine Learning**, 
-      Carl Eduard Rasmussen and Christopher K.I. Williams, MIT Press 2006. 
-      Link to an official complete PDF version of the book 
-      `here <http://www.gaussianprocess.org/gpml/chapters/RW.pdf>`_ . 
+      <http://www.gaussianprocess.org/gpml/chapters/>`_
+      **Gaussian Processes for Machine Learning**,
+      Carl Eduard Rasmussen and Christopher K.I. Williams, MIT Press 2006.
+      Link to an official complete PDF version of the book
+      `here <http://www.gaussianprocess.org/gpml/chapters/RW.pdf>`_ .
 
-.. currentmodule:: sklearn.gaussian_process
+Select Dimension Kernel
+------------------
 
+The :class:`SelectDimensionKernel` is thin wrapper around any given kernel. It
+applies the underlying kernel on a set of specific features of data. This can
+be useful when different kernels, e.g. periodic and RBF, are needed to model
+the dataset.
 
+By combining two different kinds of kernels we can construct new kernels that
+may better explain data, following example uses two kernels, :class:`ExpSineSquared`
```

Comment:
may better explain data -> are better function approximators / might model the data better?